packages/pods-core/src/v2/podConfig/AirtablePodConfig.ts

```diff
@@ -0,0 +1,55 @@
+import { AirtableConnection } from "../external-services/AirtableConnection";
+import { ExternalTarget } from "../external-services/ExternalConnectionManager";
+import { ExportPodConfigurationV2 } from "./PodV2Types";
+
+export type SrcFieldMapping =
+  | {
+      to: string;
+      type: "string" | "date" | "singleTag";
+      filter?: string;
+    }
+  | string;
+
+/**
+ * Complete Pod Config for Airtable V2
+ */
+export type AirtableV2PodConfig = ExportPodConfigurationV2 & {
+  baseId: string;
+  tableName: string;
+  sourceFieldMapping: { [key: string]: SrcFieldMapping };
+};
+
+/**
+ * This is the persisted version of the config that gets serialized into a YAML
+ * file. It must contain a reference to an airtable connection ID.
+ */
+export type PersistedAirtablePodConfig = AirtableV2PodConfig &
+  Pick<ExternalTarget, "connectionId">;
+
+/**
+ * This is the set of parameters required for actual execution of the Pod
+ */
+export type RunnableAirtableV2PodConfig = Omit<
+  AirtableV2PodConfig,
+  "podId" | "podType"
+> &
+  Pick<AirtableConnection, "apiKey">;
+
+/**
+ * Helper function to perform a type check on an object to see if it's an
+ * instance of {@link RunnableAirtableV2PodConfig}
+ * @param object
+ * @returns
+ */
+export function isRunnableAirtableV2PodConfig(
```

Comment:
why are we not using JSONSchema `ajv` validation?

Reply:
Had a discussion with @Harshita-mindfire - she's going to add ajv validation prior to pod execution, but this function is not used for that purpose - it's meant to determine whether we need to prompt the user for additional input, not to validate the correctness of the inputs at the point where it's called.